src/Tag/B.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Html\Tag;
+
+use Yiisoft\Html\Tag\Base\NormalTag;
+
+final class B extends NormalTag
```

Comment:
`<b>` is rarely used. Are you sure we need it?

Reply:
Removed.